tests/lax_numpy_test.py

```diff
@@ -97,6 +97,8 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
     op_record("abs", 1, number_dtypes, all_shapes, jtu.rand_default, ["rev"]),
     op_record("add", 2, all_dtypes, all_shapes, jtu.rand_default, ["rev"]),
     op_record("ceil", 1, float_dtypes, all_shapes, jtu.rand_default, []),
+    op_record("ceil", 1, int_dtypes + unsigned_dtypes, all_shapes,
+              jtu.rand_default, [], check_dtypes=False),
```

Comment:
Note: check_dtypes=False becuase jax always returns `jnp.float_` for integer inputs, where numpy returns the smallest float type capable of representing the largest integer of the given type.